classes/migration/upgrade/PKPv3_3_0UpgradeMigration.inc.php

```diff
@@ -334,24 +335,53 @@ private function _migrateSubmissionFiles() {
 		// Create entry in files and revisions tables for every submission_file
 		import('lib.pkp.classes.file.FileManager');
 		$fileManager = new FileManager();
-		$rows = Capsule::table('submission_files')
-			->join('submissions', 'submission_files.submission_id', '=', 'submissions.submission_id')
-			->orderBy('file_id')
-			->orderBy('revision')
+		$rows = Capsule::table('submission_files', 'sf')
+			->join('submissions AS s', 'sf.submission_id', '=', 's.submission_id')
+			->leftJoin('review_round_files AS rrf', function (JoinClause $j) {
+				$j->on('rrf.submission_id', '=', 'sf.submission_id')
+					->on('rrf.file_id', '=', 'sf.file_id')
+					->on('rrf.revision', '=', 'sf.revision');
+			})
+			->orderBy('sf.file_id')
+			->orderBy('rrf.review_round_id')
```

Comment:
Sorting by `review_round_id` here, so later on it's feasible to detect "sequence breaks".